keyboards/kyria/keymaps/gotham/config.h

```diff
@@ -0,0 +1,50 @@
+/* Copyright 2019 Thomas Baart <thomas@splitkb.com>
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#pragma once
+
+#define EE_HANDS
+#define IGNORE_MOD_TAP_INTERRUPT
+
+// Fix for Elite C rev3
+#define SPLIT_USB_DETECT
+#define NO_USB_STARTUP_CHECK
```

Comment:
This shouldn't be necessary with the above, but @zvecr can confirm.
```suggestion
```

Reply:
I will look into this as well. Thanks for the heads up!

Reply:
Yeah, you don't need this: 
https://github.com/qmk/qmk_firmware/commit/5dc91ddc60f0fd1f277345ccf20e17a8ab684cca#diff-b519ebbad91785407d49b8879eedbacc?w=1

Reply:
I just removed it.